tap_exacttarget/endpoints/content_areas.py

```diff
@@ -134,6 +138,8 @@ def sync_data(self):
                                      'ModifiedDate',
                                      content_area.get('ModifiedDate'))
 
-            singer.write_records(table, [content_area])
+            with Transformer() as transformer:
```

Comment:
@harshpatel4crest Can you make this code generic among all the streams?

Reply:
Done the code change